helm/rucio-cron-jobs/templates/cronjobs.yaml

```diff
@@ -83,5 +83,29 @@ spec:
                     secretKeyRef:
                       name: "rucio-secrets"
                       key: monit_token
+                - name: IAM_SERVER
```

Comment:
I don't think the server and client_ID need to be secret, right? We should just be able to make them parameters set directly in the Rucio-flux repository.